### PR TITLE
Refactored codes for check by collaborators. Missunderstanding fixed

### PR DESCRIPTION
**Pull request by samedkocyigit**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (FurkanKocc) [inline .vscode/launch.json:1]: please remove this file from git history. 
You can add .vscode/* to gitignore file.
Use the visual studio for better UI in unit testing

File: .vscode/tasks.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (FurkanKocc) [inline .vscode/tasks.json:1]: previous comment applies here as well.

**Review by FurkanKocc — CHANGES_REQUESTED** [review]

File: .vscode/launch.json (hunk 1 above)
Comment (furkankocyigit) [reply .vscode/launch.json:1]: please remove this file from git history.
You can add .vscode/* to gitignore file.
Use the visual studio for better UI in unit testing

File: .vscode/tasks.json (hunk 2 above)
Comment (furkankocyigit) [reply .vscode/tasks.json:1]: previous comment applies here as well.

**Review by furkankocyigit — CHANGES_REQUESTED** [review]

**Pull request merged**